hack/release-manifest.py

```diff
@@ -54,51 +61,151 @@ def create_cluster_role_yaml(csv):
     return role_data
 
 def create_clusterrolebinding_yaml():
-    # namespace = str(sys.argv[3])
-    role_binding = {
-    "apiVersion" : 'rbac.authorization.k8s.io/v1',
-    "kind" : 'ClusterRoleBinding',
-    "metadata" : {
-        "name" : 'service-binding-operator',
-    },
-    "roleRef" : {
-        "apiGroup" : 'rbac.authorization.k8s.io',
-        "kind" : 'ClusterRole',
-        "name" : 'service-binding-operator',
-    },
-    "subjects":
-        [{"kind" : 'ServiceAccount', "name": 'service-binding-operator', "namespace": 'service-binding-operator'},],
-}
-    return role_binding
+    return {
+        "apiVersion" : 'rbac.authorization.k8s.io/v1',
+        "kind" : 'ClusterRoleBinding',
+        "metadata" : {
+            "name" : 'service-binding-operator',
+        },
+        "roleRef" : {
+            "apiGroup" : 'rbac.authorization.k8s.io',
+            "kind" : 'ClusterRole',
+            "name" : 'service-binding-operator',
+        },
+        "subjects": [
+            {
+                "kind": 'ServiceAccount',
+                "name": 'service-binding-operator',
+                "namespace": 'service-binding-operator',
+            },
+        ],
+    }
 
 def create_operator_deployment(csv):
     depl = {
-    "apiVersion" : 'apps/v1',
-    "kind" : 'Deployment',
-    "metadata" : {
-        "name" : 'service-binding-operator',
-        "annotations": {"olm.targetNamespaces" : ""},
-        "namespace" : 'service-binding-operator',
-    },
-}
+        "apiVersion" : 'apps/v1',
+        "kind" : 'Deployment',
+        "metadata" : {
+            "name" : 'service-binding-operator',
+            "annotations": {"olm.targetNamespaces" : ""},
+            "namespace" : 'service-binding-operator',
+        },
+    }
+
     with open(csv, 'r') as f:
         data = yaml.load(f)
     roles = data["spec"]["install"]["spec"]["deployments"][0]
     del roles['name']
     depl.update(roles)
     return depl
 
+
+def create_cluster_issuer():
+    return {
+        "apiVersion": "cert-manager.io/v1",
+        "kind": "ClusterIssuer",
+        "metadata": {
+            "name": "selfsigned-issuer",
+        },
+        "spec": {
+            "selfSigned": {}
+        }
+    }
+
+def create_webhook_certificate():
+    return {
+        "apiVersion": "cert-manager.io/v1",
+        "kind": "Certificate",
+        "metadata": {
+            "name": "service-binding-operator-serving-cert",
+            "namespace": "service-binding-operator",
```

Comment:
can we create a global var with `service-binding-operator` namespace and use it in the script - it appears in some many places already.